x/subspaces/simulation/decoder.go

```diff
@@ -60,6 +60,18 @@ func NewDecodeStore(cdc codec.Codec) func(kvA, kvB kv.Pair) string {
 			cdc.MustUnmarshal(kvB.Value, &sectionB)
 			return fmt.Sprintf("SectionA: %s\nSectionB: %s\n", &sectionA, &sectionB)
 
+		case bytes.HasPrefix(kvA.Key, types.UserAllowancePrefix):
+			var grantA, grantB types.Grant
+			cdc.MustUnmarshal(kvA.Value, &grantA)
+			cdc.MustUnmarshal(kvB.Value, &grantB)
+			return fmt.Sprintf("UserGrantA: %s\nUserGrantB: %s\n", &grantA, &grantB)
+
+		case bytes.HasPrefix(kvA.Key, types.GroupAllowancePrefix):
+			var grantA, grantB types.Grant
+			cdc.MustUnmarshal(kvA.Value, &grantA)
+			cdc.MustUnmarshal(kvB.Value, &grantB)
+			return fmt.Sprintf("GroupGrantA: %s\nGroupGrantB: %s\n", &grantA, &grantB)
```

Comment:
I think these cases can be merged properly since they are both dealing with a `Grant` object:
```suggestion
		case bytes.HasPrefix(kvA.Key, types.UserAllowancePrefix):
		case bytes.HasPrefix(kvA.Key, types.GroupAllowancePrefix):
			var grantA, grantB types.Grant
			cdc.MustUnmarshal(kvA.Value, &grantA)
			cdc.MustUnmarshal(kvB.Value, &grantB)
			return fmt.Sprintf("GrantA: %s\nGrantB: %s\n", &grantA, &grantB)
```